### PR TITLE
Add services.ini, and update README

### PR DESCRIPTION
**Pull request by briandds**
This PR adds a `services.ini` which has a list of the services managed by this repo. It also updates the README with a bit more information.

It also updates `script/test` to use log files and not exit if a single test exits.

### CODE REVIEW

File: script/test [hunk 1]
```diff
@@ -1,10 +1,21 @@
 #!/bin/bash -e
 
-SERVICES="atst authnid fundz"
-cd $(dirname $0)/../services
+cd $(dirname $0)/..
+source services.ini
+cd services
```
Comment (jordangov) [inline script/test:5]: This won't work if the directory doesn't exist yet, right? We need to either create the directory, or not git ignore it.

Comment (jordangov) [inline script/test]: This won't work if the directory doesn't exist yet, right? We need to either create the directory, or not git ignore it.

**Review by jordangov — CHANGES_REQUESTED** [review]

File: script/test (hunk 1 above)
Comment (jordangov) [reply script/test:5]: Ah... I see it's in the other script... hard to tell, maybe a comment here?

Comment (briandds) [reply script/test:5]: How about this: 5003167?

Comment (jordangov) [reply script/test:5]: Aaah... yeah, that'll work.

**Review by jordangov — APPROVED** [review]

**Pull request merged**